src/settings.js

```diff
@@ -2060,6 +2060,11 @@ var RUNTIME_DEBUG = false;
 // library symbol.
 var LEGACY_RUNTIME = false;
 
+// User-defined functions to wrap with signature conversion
+// example use -sSIGNATURE_CONVERSIONS=['someFunction:_p']
+// [link]
+var SIGNATURE_CONVERSIONS = [];
```

Comment:
I've trying to think of some way that we can embed this in the source code itself.

Something like `__attribute__(emcc_sig("_p"))`.  

Perhaps we could add this setting in the interim?

Reply:
Do you want to somehow wrap them automatically with that? Will check what I can do